docs/core/logger.md

```diff
@@ -604,7 +604,28 @@ By default all registered loggers will be modified. You can change this behavior
 ---8<-- "examples/logger/src/cloning_logger_config.py"
 ```
 
-**What's the difference between `append_keys` and `extra`?**
+### How can I add standard library logging attributes to a log record?
+
+The Python standard library log records contains a [large set of atttributes](https://docs.python.org/3/library/logging.html#logrecord-attributes){target="_blank"}, however only a few are included in Powertools Logger log record by default.
```

Comment:
Nit, but.... "Powertools `Logger`" instead of `Powertools Logger"? 

```suggestion
The Python standard library log records contains a [large set of atttributes](https://docs.python.org/3/library/logging.html#logrecord-attributes){target="_blank"}, however only a few are included in Powertools `Logger` log record by default.
```

Reply:
I see why it'd make sense, but it'd be difficult to stay consistent over time - let's keep Powertools Logger. At least until we explore tools like [Vale](https://vale.sh/).

Reply:
Sounds good. I think this is ready to merge then.